base58/README.md

```diff
@@ -4,16 +4,22 @@ Bitcoin base58 encoding
 This crate provides encoding and decoding of base58 strings as defined by the Bitcoin ecosystem
 including the checksum.
 
-There are two other crates on crates.io that implement base58 encoding and decoding. This
-crate differs from them because:
+There are at least three other crates on crates.io that implement base58 encoding and decoding.
+
+This crate differs from them because:
 
 1. [bitcoin-base58](https://crates.io/crates/bitcoin-base58) is transpiled from the C++ code in
    Bitcoin Core as part of a large long-term transpilation project; this crate is a pure Rust
    implementation intended to be production-ready and to provide an Rust-idiomatic API.
 
-2. [base58](https://crates.io/crates/base58) is incomplete and appears unmaintained as of
-   February 2024. It does not validate checksums and will therefore accept invalid Bitcoin
-   addresses. It may be appropriate in cases where performance is more important than safety.
+2. [base58](https://crates.io/crates/base58) implements parsing but does not validate checksums (see
+   (3)). It may be appropriate in cases where performance is more important than safety.
+
+3. [base58check](https://crates.io/crates/base58check) depends on
+   `base58` and [sha2](https://crates.io/crates/sha2/0.8.2)
```

Comment:
And appears unmaintained.

Reply:
"Appears unmaintained" has been put back in this PR - I believe this thread is resolved.